lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -179,6 +186,38 @@ const Reportback = {
     });
   },
 
+  /**
+   * Get Kudos Reactions for a Reportback item.
+   *
+   * @param  {array} data  Array of Kudos objects from API.
+   * @return {array}
+   */
+  getReactions: function(data) {
+    console.log(data);
+    let reactions = [];
+
+    for (let i = 0, count = this.reactions.approved.length; i < count; i++) {
```

Comment:
@DFurnes was considering pulling in [lodash's `forEach`](https://lodash.com/docs#forEach) for some of these loops I have. Thoughts?